docs/Cube.js-Backend/Environment-Variables-Reference.md

```diff
@@ -15,10 +15,10 @@ behavior. Some of these variables can also be set via [configuration options][li
 |Environment variable                |Used With    |Description                                                                                                                                                     |Possible Values                                          |
 |------------------------------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|
 |`CUBEJS_CACHE_AND_QUEUE_DRIVER`     |Caching      |The cache and queue driver to use for the Cube.js deployment. Defaults to `redis`                                                                               |`redis | memory`                                         |
-|`CUBEJS_DEV_MODE`                   |Configuration|If `true`, enabled debug mode. Defaults to `false`.                                                                                                             |`true | false`                                           |
+|`CUBEJS_DEV_MODE`                   |Configuration|If `true`, enables debug mode. Defaults to `false`.                                                                                                             |`true | false`                                           |
```

Comment:
@hassankhan @ovr why it says 'debug' mode instead of dev mode?

Reply:
@keydunov good catch, my miss..